### ISSUE TITLE
Strings containing # break the web notebook

### ISSUE THREAD

**Issue by oxinabox** [google-research/dex-lang#310]
This one is weird.
Here is a MWE
1. Create a file, give it the content `"#"` (optionally put a new line, or really anything else before or after)
2. start the web notebook on that file, and open a web browser  to it
3. Notice that the page is blank and the terminal running the web repl is now displaying:
```
Streaming output to localhost:8000
[]
["dynamic.js"]
["getnext"]
1:3:
  |
1 | "#"
  |   ^
unexpected '"'
expecting '&', '?', '|', or lowercase letter

CallStack (from HasCallStack):
  error, called at src/lib/RenderHtml.hs:75:30 in dex-0.1.0.0-CA4uXQ86C1yKMObiJn1rPk:RenderHtml
["sw.js"]
```

I note that the same thing works fine in the normal REPL
```
>=> "#"
['#']
>=>
```
I can even `include` the file that broke the web repl without issue.

**Comment by danieldjohnson** [comment]
This is probably a bug in the syntax highlighter, which only runs in notebook mode.

**Comment by dougalm** [comment]
Yes, we should fix the syntax highlighter. We should also fall back to printing plain strings instead of [throwing a hard error](https://github.com/google-research/dex-lang/blob/main/src/lib/RenderHtml.hs#L74) when it fails, since it's purely cosmetic.
